Assets/Scripts/Explosion.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class Explosion : MonoBehaviour {
+
+	[SerializeField]
+	private float explosionSize;
+	[SerializeField]
+	private float visualExpansionRate;
+	[SerializeField]
+	private float colliderExpansionRate;
+
+	private SphereCollider sphereCollider;
+	private Vector3 expansionVector;
+
+	// Use this for initialization
+	void Start () {
+		sphereCollider = GetComponent<SphereCollider>();
```

Comment:
you arne't guaranteeing that this won't be null. You need to do two things:

1. `RequireComponent`
2. Assert that it is not null